content/library/2020-04-08-understand-cpu-arch.md

```diff
@@ -0,0 +1,17 @@
+---
+title: ""
```

Comment:
Everything looks good - just one little thing: could you please add the human-readable title of the video into this field?

Reply:
fixed